### PR TITLE
Add: support specifying cipher suites in tls connection #3019

### PR DESCRIPTION
**Pull request by clock21am**
## Which problem is this PR solving?
- Add  support specifying cipher suites in tls connection #3019

## Short description of the changes
- Add cipher parameter in tls config

### CODE REVIEW

**Comment by clock21am** [comment]
@yurishkuro Let me know what do you think?

File: pkg/config/tlscfg/options.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
+	"github.com/coreos/etcd/pkg/tlsutil"
```
Comment (yurishkuro) [inline pkg/config/tlscfg/options.go:21]: I don't think we want an external dependency for such minor functionality. Also, I am not seeing this package in https://github.com/etcd-io/etcd/tree/main/pkg

Comment (clock21am) [reply pkg/config/tlscfg/options.go]: @yurishkuro removed this external dependency and made it a sample. Hope this makes sense.

Comment (clock21am) [reply pkg/config/tlscfg/options.go]: as it was mentioned in this document  https://docs.fluentd.org/configuration/transport-section#tls-setting do you want to provide some default cipher in the repo itself?

Comment (clock21am) [reply pkg/config/tlscfg/options.go]: and one more doubt why naming ciphers if want to pass just string ? are we expecting multiple ciphers ?

**Comment by yurishkuro** [comment]
I have questions on the original ticket.

**Comment by jpkrohling** [comment]
What's the status of this PR?

**Comment by clock21am** [comment]
@jpkrohling there were questions on the original tickets have to look but haven’t got the chance can take a look in the evening though

**Comment by yurishkuro** [comment]
Already addressed by #3564

**Pull request closed without merging**